controllers/ebpf/agent_controller.go

```diff
@@ -79,14 +79,15 @@ const (
 )
 
 const (
-	exportKafka        = "kafka"
-	exportGRPC         = "grpc"
-	kafkaCerts         = "kafka-certs"
-	averageMessageSize = 100
-	bpfTraceMountName  = "bpf-kernel-debug"
-	bpfTraceMountPath  = "/sys/kernel/debug"
-	bpfNetNSMountName  = "var-run-netns"
-	bpfNetNSMountPath  = "/var/run/netns"
+	exportKafka                = "kafka"
+	exportGRPC                 = "grpc"
+	kafkaCerts                 = "kafka-certs"
+	averageMessageSize         = 100
+	bpfTraceMountName          = "bpf-kernel-debug"
+	bpfTraceMountPath          = "/sys/kernel/debug"
+	bpfNetNSMountName          = "var-run-netns"
+	bpfNetNSMountPath          = "/var/run/netns"
+	droppedFlowsAlertThreshold = 100
```

Comment:
the threshold of 100 , is it based on some testing or an arbitrary value that we think is good (not too high or not too low)?

Reply:
random number atm

Reply:
That's arbitrary, but I don't see how to not make it arbitrary, unless by making it a configurable threshold. People who would like to set another limit can still copy/paste the rule to a new one, and turn off this alert in netobserv. The rule is still useful in that sense.

Still, I did test it on my cluster.
With my "medium" hey-ho test intensity (which actually is quite high already), I'm between 10 and 15 drops/sec. With my "high" intensity I'm around 25-30.

Bigger clusters would likely see more drops/sec under stress.

Reply:
got it, thanks!